source/RedisInside.Tests/RedisInside.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net451;net46;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>

```

Comment:
If we remove net46 this can not be used from Full framework < net471 without dependency on netstandard

Reply:
Well this can not work on linux if we keep the old .NetFramework

Reply:
What do you mean? It’s multi targeting. It will build for all the listed frameworks. The nuget package will contain both builds and your project will pick up the correct one

Reply:
Sorry I'm pretty new to dotnet so I might say dumb things.
But this implementation is not working with old frameworks. I tried re-adding them and tests are failing on those. I'm guessing they are not able to run linux scripts properly.
If this is mandatory for you to keep old frameworks, I will just use a fork of your project. However, if people want to use a cross platform redis-inside, I don't think that keeping the compatibility is a necessity.

Reply:
I’m pretty confident it’s possible to make this work cross platform and keeping the net46 support. The packaging needs to be done on Windows and some tests but that’s why we have conditionals 🙂I don’t have time tonight but I can have a look tomorrow

Reply:
Thanks ! I only have linux systems availables so it's a bit hard for me to evaluate this on windows 😄

Reply:
Hey, Have you had the time to check the compatibility with old frameworks ?